### ISSUE TITLE
What are the issues to which contributors can contribute?

### ISSUE THREAD

**Issue by GaurangTandon** [internetarchive/wayback-machine-chrome#274]
Hi,

I notice that [commits](https://github.com/internetarchive/wayback-machine-chrome/commits/master) are being made almost on a daily frequency to this repo. That means a lot of stuff has been done and has to be done. I feel I could have helped in several of these tasks, like, refactoring the JS files, adding donation buttons, etc. 

The [issues](https://github.com/internetarchive/wayback-machine-chrome/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) that are there currently are months old, and many of them have become obsolete ([example](https://github.com/internetarchive/wayback-machine-chrome/issues/6)) 

Could the maintainer please create a few issues describing what are the tasks they need help with next, to which a contributor like me can contribute?

**Comment by anishsarangi** [comment]
You can start with refactoring the JS files.
Thank you.

**Comment by GaurangTandon** [comment]
Hi @anishsarangi , thanks for your reply. 

I have a question: Do you use a style guide, like `.editorconfig`, or `.prettierrc`? There are chances that our tab style/quote style/function style/etc. might not match, and I would be glad to have consistency throughout the repo. If you want should I create a basic `.prettierrc`/`.editorconfig` file? Once that is done, and once the maintainers confirm if the style specifications are ok with them, I can also run all the js files one by one through this style guide, so they all are consistent, and create a PR for the same. Thoughts?

**Comment by mkfeuhrer** [comment]
@anishsarangi @vbanos could you provide with some future features or issues on which contributions can be made. Can't find any relevant issues. If there is could you point me to one. Thanks.

**Comment by daveloyall** [comment]
The two most recent committers are @MaxReinisch and @vbanos.  Hello!

A user on IRC asked:

> Who do I contact regarding contributions to the Wayback Machine Chrome Extension?                                                                                                                                                                            

I replied:

> I am not part of the archive.org team.  But, I can tell you that this source code repository is active:  https://github.com/internetarchive/wayback-machine-chrome
> There do seem to be some issues and pull requests there.

The user replied:

> There are issues and PRs, yeah. But, there is no response from the people who are actively committing.                                                                                                                                                    

Maybe it is time for some kind of contributors' guide document?  Or for a review of tickets?  Or to grant some additional person permission to merge PRs or close issues?

Keep up the good work.  Cheers!

**Comment by radicalswetank** [comment]
I agree with @GaurangTandon @mkfeuhrer @daveloyall 
@MaxReinisch @vbanos  @anishsarangi Please Reply

**Comment by radicalswetank** [comment]
@daveloyall Can you give me the IRC link?

**Comment by anishsarangi** [comment]
I have addressed in #299 .
Thank you.
